sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -21,23 +21,19 @@ documentation][event_hubs_product_docs] | [Samples][sample_examples]
 
 ## Table of contents
 
-- [Azure Event Hubs client library for Java](#azure-event-hubs-client-library-for-java)
-  - [Table of contents](#table-of-contents)
-  - [Getting started](#getting-started)
-    - [Prerequisites](#prerequisites)
-    - [Include the package](#include-the-package)
-    - [Authenticate the client](#authenticate-the-client)
-  - [Key concepts](#key-concepts)
-  - [Examples](#examples)
-    - [Publish events to an Event Hub](#publish-events-to-an-event-hub)
-    - [Consume events from an Event Hub partition](#consume-events-from-an-event-hub-partition)
-    - [Consume events using an EventProcessorClient](#consume-events-using-an-eventprocessorclient)
-  - [Troubleshooting](#troubleshooting)
-    - [Enable client logging](#enable-client-logging)
-    - [Enable AMQP transport logging](#enable-amqp-transport-logging)
-    - [Exceptions](#exceptions)
-  - [Next steps](#next-steps)
-  - [Contributing](#contributing)
+- [Table of contents](#table-of-contents)
```

Comment:
Silly question, does the table of contents need a link to the table of contents? Seems a little redundant

Reply:
LOL. I have an auto ToC generator... I'll remember to remove it.